Makefile

```diff
@@ -196,6 +196,10 @@ unit-race:
 	@$(call print, "Running unit race tests.")
 	env CGO_ENABLED=1 GORACE="history_size=7 halt_on_errors=1" $(UNIT_RACE)
 
+unit-bench: $(BTCD_BIN)
+	@$(call print, "Running benchmark tests.")
+	$(UNIT_BENCH)
```

Comment:
I think it would also be good to run `make unit-bench` in CI.

I think the file we need to modify for that is `.github/workflows/main.yml`

Reply:
thanks for the reminder, I added it.

Reply:
I don't see what the benefit of running the benchmark tests in CI is? This takes up another slot of our already pretty scarce action runner slots. And as long as we don't manually compare the results with previous runs, this wouldn't show any performance degradation.

Reply:
Removed it from the CI